clients/src/main/java/org/apache/kafka/common/metrics/Metrics.java

```diff
@@ -171,6 +195,7 @@ public void removeSensor(String name) {
                     }
                 }
             }
```

Comment:
Resolved offline with Joel. Given the large expiration times, this is unlikely to be an issue.